dictionary.json

```diff
@@ -17750,10 +18445,24 @@
     "fields": []
   },
   {
-    "toaq": "to",
-    "type": "separator",
-    "english": "helper particle in forethought conjunctions: to + conj ▯ to ▯",
-    "gloss": "CONJ",
+    "toaq": "tofu",
+    "type": "predicate",
+    "english": "▯ is tofu.",
+    "gloss": "tofu",
+    "short": "",
+    "keywords": [],
+    "frame": "a",
+    "distribution": "d",
+    "pronominal_class": "maq",
+    "notes": [],
+    "examples": [],
+    "fields": []
+  },
+  {
+    "toaq": "tó",
+    "type": "focus prefix",
+    "english": "only",
+    "gloss": "only",
     "short": "",
     "keywords": [],
     "notes": [],
```

Comment:
Below this: Toaq should be capitalized according to the newly clarified orthography rules